flutter/microbenchmarks/lib/src/jni_bench.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:ffi';
+import 'dart:math';
+
+import 'package:jni/jni.dart';
+import 'package:benchmarking/benchmarking.dart';
+import 'package:flutter/foundation.dart';
+
+Future<void> execute() async {
```

Comment:
Out of curiousity, have you measured the perfomance improvement between the previous way of using method channels vs. using jni?

Reply:
Not per se. I have profiled it previously and with the new code and the biggest benefit we get is that we can now use `Bitmap` directly from raw RGBA so we don't have to convert to PNG (which is in hundreds of milliseconds, even though it's async) and to write an intermediary file (two IO ops - write & read). So there really is no one-to-one comparison between the approaches.
Another benefit was being able to move the work to another isolate altogether. Actually, I need to check whether I can pass an Uint8List efficiently between isolates. Last time I checked it didn't work.